backend/models/planInvite.ts

```diff
@@ -0,0 +1,65 @@
+import { type PlanInvite } from '@prisma/client'
+import prisma from '../utils/prismaClient'
+const db = prisma
+
+export default class PlanInviteModel {
+  private static getNthDateFromToday(numOfDay: number): Date {
+    const today = new Date()
+    const nthDayFromToday = new Date(today)
+    nthDayFromToday.setDate(nthDayFromToday.getDate() + numOfDay)
+
+    return nthDayFromToday
+  }
```

Comment:
This seems like a nice utility function! It's okay to leave it here if you'd like, but a common thing is to create a utility folder / module so other people can use this function.